recipes/examl/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: "examl"
+  version: "3.0.17"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: v3.0.17.tar.gz
+  url: https://github.com/stamatak/ExaML/archive/v3.0.17.tar.gz
+  md5: "bf1e2c3774646d3793fdc8a4c9145af7"
+
+requirements:
+  build:
+    - mpich
```

Comment:
I'm giving this a last shot for the osx build. Unfortunately, I just realized that my erroneous push of another feature branch to master has resulted in two packages being built here - see PR #2690 for rpkmforgenes. Since I already pushed to my forked repo, rpkmforgenes has already been merged with master. I don't know if this is problem for the bioconda commit history once PR #2690 is merged. One option would be that I fork bioconda anew and resubmit these pull requests based on a correct bioconda history. What do you think?